ssl/config/gradle.properties

```diff
@@ -0,0 +1,2 @@
+io.deephaven.project.ProjectType=JAVA_PUBLIC
+languageLevel=8
```

Comment:
why 8? would it make sense to document this - and should ssl/ (and possibly children) get a readme documenting what they are intended to be for etc?

Reply:
Anything that the java-client depends on needs to be scoped to language level 8.